src/pages/help-centre/deriv-x.js

```diff
@@ -135,7 +156,14 @@ const ResetDerivXPassowrd = () => (
         <Text>
             <Localize
                 translate_text="Go to your <0>Account settings</0>. Under “Security and safety”, select “Passwords”. You can reset your Deriv X password under “Trading password”."
-                components={[<LocalizedLinkText color="red" key={0} />]}
+                components={[
+                    <ExternalLink
+                        href={`${deriv_x_app_url}/account-settings`}
```

Comment:
This should be pointing to `https://app.deriv.com/account/personal-details`. You can use the `deriv_app_url` for this

Reply:
okay thanks